hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -331,11 +332,11 @@ protected HStore(final HRegion region, final ColumnFamilyDescriptor family,
       confPrintThreshold = 10;
     }
     this.parallelPutCountPrintThreshold = confPrintThreshold;
-    LOG.info("Store={},  memstore type={}, storagePolicy={}, verifyBulkLoads={}, "
-            + "parallelPutCountPrintThreshold={}, encoding={}, compression={}",
+    LOG.info("Created HStore: Store={},  memstore type={}, storagePolicy={}, verifyBulkLoads={}, "
```

Comment:
Why have 'HStore:' in here? Instead of 'Created HStore: Store=', just have 'Created '.

Reply:
> Why have 'HStore:' in here? Instead of 'Created HStore: Store=', just have 'Created '.

Got it